node/DEVELOPER.md

```diff
@@ -156,6 +156,7 @@ In order to run these tests, use:
 ```bash
 npm run test-modules -- --cluster-endpoints=<address>:<port>
 ```
+
```

Comment:
is this necessary?

Reply:
No, added by linter (`npm run lint:fix`)